src/DriverManager.php

```diff
@@ -60,8 +62,8 @@
  *     primary?: OverrideParams,
  *     replica?: array<OverrideParams>,
  *     serverVersion?: string,
- *     sharding?: array<string,mixed>,
- *     slaves?: array<OverrideParams>,
+ *     sessionMode?: int,
+ *     url?: string,
```

Comment:
`url` should be gone as per #5850.

Reply:
That's a nice improvement. In this case, I'll need to rework this test: https://github.com/doctrine/dbal/blob/95d093a306e3cd059a5ddf2e461c24c6fddf5a4f/tests/Logging/MiddlewareTest.php#L35-L39